examples/ads.amp.html

```diff
@@ -854,6 +855,14 @@ <h2 id="nativo">Nativo</h2>
       data-request-url="http://localhost:9876">
   </amp-ad>
 
+  <h2 id="navegg">Navegg</h2>
+  <amp-ad width=320 height=50
+      type="navegg"
+      data-acc="XXXXX"
```

Comment:
the examples/ads.amp.html is meant for demo and testing. Can you put put a fake ad example here, so that we know the navegg loads correctly.